jpegenc/subblocks/huffman/ac_cr_rom.py

```diff
@@ -0,0 +1,775 @@
+"""The above module is a Luminance Rom
+    for AC component for huffman module"""
+
+from myhdl import always_seq, block, Signal, ResetSignal
+from myhdl import intbv, always_comb, concat
+from myhdl.conversion import analyze
+
+
+@block
+def ac_cr_rom(clock, reset, runlength, vli_size, vlc_ac_size, vlc_ac):
+    """HDL implementation of Chrominance AC ROM"""
+
+    width_runlength = len(runlength)
+    width_size = len(vli_size)
+
+    rom_addr = Signal(intbv(0)[(width_runlength+width_size):])
+
+    @always_comb
+    def assign1():
+        """assign rom address"""
+
+        rom_addr.next = concat(runlength, vli_size)
+
+    @always_seq(clock.posedge, reset=reset)
+    def assign():
+        """storing variable length coded data in rom"""
+        if runlength == 0:
+
+            if vli_size == 0x0:
```

Comment:
@cfelton . We do not know the probabilities assigned by `standard huffman tables` for each pixel.
I have added that as an issue in my repo.

It still needs some research to develop the algorithm(knowing about pixel probability). I will go through the `Huffman decoder`, it may give an insight of this.